query-connector/src/app/database-service.ts

```diff
@@ -89,17 +90,19 @@ export const mapQueryRowsToValueSetItems = async (rows: QueryResultRow[]) => {
 };
 
 /*
- * The expected return type from the eRSD API.
+ * The expected return type from both the eRSD API and the VSAC FHIR API.
  */
-type ErsdResponse = Bundle | OperationOutcome;
+type ErsdOrVsacResponse = Bundle | OperationOutcome;
 
 /**
  * Fetches the eRSD Specification from the eRSD API. This function requires an API key
  * to access the eRSD API. The API key can be obtained at https://ersd.aimsplatform.org/#/api-keys.
  * @param eRSDVersion - The version of the eRSD specification to retrieve. Defaults to v2.
  * @returns The eRSD Specification as a FHIR Bundle or an OperationOutcome if an error occurs.
  */
-export async function getERSD(eRSDVersion: number = 2): Promise<ErsdResponse> {
+export async function getERSD(
+  eRSDVersion: number = 2,
```

Comment:
I think @m-goggins' recommendation was to bump this to 3. not really sure it needs its own PR 😅